### PR TITLE
Support for db.run tracing

### PR DESCRIPTION
**Pull request by schiwekM**
Things like
`await db.run('SELECT ID FROM AdminService_Books')`
or
`await cds.run(`Call "getTranscriptOfRecordsColumns"(pUserID => ?)`, [req.user.id]);`

should now be supported.

### CODE REVIEW

**Comment by sjvans** [comment]
hi @schiwekM 

thanks!

in https://github.com/cap-js/telemetry/pull/195, i changed to `db - ${arg.query}` in case `arg.event === undefined`, which leads to
```
[telemetry] - elapsed times:
    0.00 →   1.43 =   1.43 ms  db - SELECT ID FROM AdminService_Books
    0.65 →   0.96 =   0.30 ms    @cap-js/sqlite - prepare SELECT ID FROM AdminService_Books
    1.32 →   1.38 =   0.05 ms    @cap-js/sqlite - stmt.all SELECT ID FROM AdminService_Books
```
and 
```
[telemetry] - elapsed times:
    0.00 →   1.43 =   1.43 ms  db - Call "getTranscriptOfRecordsColumns"(pUserID => ?)
    0.65 →   0.96 =   0.30 ms    @cap-js/sqlite - prepare SELECT ID FROM AdminService_Books
    1.32 →   1.38 =   0.05 ms    @cap-js/sqlite - stmt.all SELECT ID FROM AdminService_Books
```
in your test, which i think is fine. hence, i'd remove what you do in the try block. any objections?

however, i think we should probably ensure that the span name doesn't get too long...

**Comment by schiwekM** [comment]
Fine by me. Just wondering if there is a different way for keeping the span name short, but on the other side there shouldn't be so many db.run string executions so it probably does not really matter.

File: lib/tracing/trace.js [hunk 1]
```diff
@@ -42,7 +42,7 @@ function _getParentSpan() {
 }
 
 function _getSpanName(arg, fn, targetObj) {
-  const { phase, event, path } = arg
+  let { phase, event, path } = arg
```
Comment (sjvans) [inline lib/tracing/trace.js:45]:
```suggestion
  const { phase, event, path } = arg
```

File: test/tracing.test.js [hunk 2]
```diff
@@ -47,7 +47,7 @@ describe('tracing', () => {
     // 2: action + spawned action
     expect(log.output.match(/\[telemetry\] - elapsed times:/g).length).to.equal(2)
   })
-
+
```
Comment (sjvans) [inline test/tracing.test.js:50]:
```suggestion

```

File: test/tracing.test.js [hunk 3]
```diff
@@ -47,7 +47,6 @@ describe('tracing', () => {
     // 2: action + spawned action
     expect(log.output.match(/\[telemetry\] - elapsed times:/g).length).to.equal(2)
   })
-
   describe('db', () => {
```
Comment (sjvans) [inline test/tracing.test.js:50]:
```suggestion

  describe('db', () => {
```

**Review by sjvans — APPROVED** [review]

**Pull request merged**